lib/database.js

```diff
@@ -435,28 +435,33 @@ module.exports = {
   },
 
   get_last_txs_ajax: function(start, length, min, cb) {
-      Tx.countDocuments({'total': {$gte: min}}, function(err, count){
-        Tx.find({'total': {$gte: min}}).sort({blockindex: 'desc'}).skip(Number(start)).limit(Number(length)).exec(function(err, txs){
-          if (err) {
-            return cb(err);
-          } else {
-            return cb(txs, count);
-          }
-        });
+    lib.get_blockcount(function(blockcount) {
```

Comment:
@typhoonsimon As per the function's name "get_last_txs_ajax" - this function does NOT show latest blocks - it shows **Latest Transactions**, we cannot use the blockcount call here.

This does look very efficient though and would work almost as is for an additional function for latest blocks, which we could then set in the config for the user to display latest TX or latest BLOCKS on the front page.

In terms of making this code work for latest TXs what's the performance difference of Tx.countDocuments vs Tx.count here?

tx_count might be a stat we should keep updated via block indexing?